Contextualizing preprints in motion (#64)
greenelab/annorxiver_manuscript: content/03.methods.md

We trained this model on document embeddings of PMC articles that did not contain a matching preprint version.
We used the following parameters to train the model: a hidden size of 2, a learning rate of 0.001, lambda_b of 0, lambda_c of 0.001, and lambda_d of 0.001 for 2000 iterations.
When a user requests a new document, we can then project that document onto our generated two-dimensional space; thereby, allowing the user to see where their preprint falls along the landscape.
We illustrate our recommendations as a short list and provide access to our network visualization at our website (see Software and Data Availability).

> ### Contextualizing Preprints in Motion

Reviewer: ```suggestion
### Analysis of the Preprints in Motion Collection
```

> During the initial stages of the COVID-19 pandemic, repositories such as bioRxiv and medRxiv received an influx of preprint submissions.

Reviewer: ```suggestion
Our manuscript describes the large-scale analysis of bioRxiv.
Concurrent with our work, another set of authors performed a detailed curation and analysis of a subset of bioRxiv [@doi:10.1101/2021.02.20.432090] that was focused on preprints posted during the initial stages of the COVID-19 pandemic.
```

> This influx along with prior skepticism on preprint efficacy [@doi:10.1101/2020.05.22.111294] prompted researchers to analyze preprint to publication changes in preprints posted within January 1st, 2020 to April 30th, 2020 timeframe [@doi:10.1101/2021.02.20.432090]. 

Reviewer: ```suggestion
The curated analysis was designed to examine preprints at a time of increased readership [@doi:10.1101/2020.05.22.111294] and includes certain preprints posted from January 1st, 2020 to April 30th, 2020 [@doi:10.1101/2021.02.20.432090]. 
```

> We sought to contextualize this subset within our global picture of the bioRxiv preprint landscape.

Reviewer: ```suggestion
We sought to contextualize this subset, which we term "Preprints in Motion" after the title of the preprint [@doi:10.1101/2021.02.20.432090], within our global picture of the bioRxiv preprint landscape.
```

> We extracted all preprint digital object identifiers (DOI) within [@doi:10.1101/2021.02.20.432090] and filtered out entries that corresponded to preprints within the medRxiv repository.

Reviewer: ```suggestion
We extracted all preprints from the set reported in Preprints in Motion [@doi:10.1101/2021.02.20.432090] and retained any entries in the bioRxiv repository.
```

> We manually downloaded the XML version of the remaining preprints and mapped their corresponding DOIs with their published counterparts using the same method mentioned in the Mapping bioRxiv Preprints to their Published Counterparts section. 

Reviewer: ```suggestion
We manually downloaded the XML version of these preprints and mapped them to their published counterparts as described above. 
```

We used Pubmed Central's DOI converter [@url:https://www.ncbi.nlm.nih.gov/pmc/pmctopmid/] to map the published article DOIs with their respective PubMed Central IDs.

> We filtered out published articles that weren't a part of the PMCOA corpus and performed a token analysis that compared all preprints against their published versions (see Corpora Comparison).

Reviewer: ```suggestion
We retained articles that were included in the PMCOA corpus and performed a token analysis as described to compare these preprints with their published versions.
```

> Next, we generated document embeddings (see Constructing a Document Representation for Life Sciences Text) for every obtained preprint and published article.

Reviewer: ```suggestion
As above, we generated document embeddings for every obtained preprint and published article.
```

> We projected these preprint embeddings onto our publication landscape to visually observe the dispersion of this subset (see Web Application for Discovering Similar Preprints and Journals).

Reviewer: ```suggestion
We projected these preprint embeddings onto our publication landscape to visually observe the dispersion of this subset.
```

> Finally, we performed a time analysis to observe how peer review changes affect a preprint's time to publication (see Measuring Time Duration for Preprint Publication Process).   

Reviewer: ```suggestion
Finally, we performed a time analysis that paralleled our approach for the full set of preprint-publication pairs to examine relationships between linguistic changes and the time to publication.   
```
